public/javascripts/libraryTable.js

```diff
@@ -42,17 +44,17 @@ var libraryTableComponent = (function() {
                                     }()
                                 ]),
                                 m('td', [
-                                  function() {
-                                      if (s.private) {
-                                          return m('a', {
-                                              href: '/user/privatesong/' + s._id
-                                          }, s.title)
-                                      } else {
-                                        return m('a', {
-                                            href: '/' + s._id
-                                        }, s.title)
-                                      }
-                                  }()
+                                    function() {
+                                        if (s.private) {
+                                            return m('a', {
+                                                href: '/user/privatesong/' + s._id
```

Comment:
question: what happens if someone visits a private song but is not logged in as the user who owns it. What do they see?